### ISSUE TITLE
Using a single component to add any icons without importing all

### ISSUE THREAD

**Issue by alic-xc** [rendinjast/iconsax-react#13]
Hi,

Thanks for this great package. I would like to ask if there is a component to use that prevents us from importing any icons we want to use.

`
<Icon name="family" size={20} />

`

**Comment by Chinedu19** [comment]
Hi, this issue description isn't clear. Can you rephrase so others can understand?

**Comment by alic-xc** [comment]
> Hi, this issue description isn't clear. Can you rephrase so others can understand?

Instead of importing any components, can't we just have a single component that we can use to specify any font icon?

**Comment by Samandcode** [comment]
> > Hi, this issue description isn't clear. Can you rephrase so others can understand?
> 
> Instead of importing any components, can't we just have a single component that we can use to specify any font icon?

Exacly

A feature like that would be helpful when one is getting these icon from an array of items

**Comment by Samandcode** [comment]
I’ll check it out 👏🏼👏🏼

**Comment by yamilelias** [comment]
Hey @esamueljsx! Our company is updating the package because there are several issues we need to address to continue using it. So, feel free to check it out! We just made a huge improvement on the bundle size. You can now import them like this:

```jsx
// Keeps the runtime variant prop and includes all six SVG variants.
import { EmojiHappy } from '@weprograpp/iconsax-react/icons/EmojiHappy';

// Smallest bundle: includes only the Linear SVG variant.
import { EmojiHappy as BulkEmojiHappy } from '@weprograpp/iconsax-react/linear/EmojiHappy';
```

Give it a try and feel free to add any issues there. We'll try our best to keep improving this awesome package! 🙌 You can check it out if you want here:

https://www.npmjs.com/package/@weprograpp/iconsax-react

Please feel free to leave anything on our fork so we can improve it!
